plugins/inlinetoolbar/plugin.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+( function() {
+	'use strict';
+
+	CKEDITOR.plugins.add( 'inlinetoolbar', {
+		requires: 'balloonpanel',
+		init: function( editor ) {
+			CKEDITOR.ui.inlineToolbar.prototype = CKEDITOR.tools.extend( CKEDITOR.ui.balloonPanel.prototype );
+			CKEDITOR.ui.inlineToolbar.prototype.build = function() {
```

Comment:
Why, copy and pasting stuff? Keep it DRY. You want to reuse original method + apply any adjustments before / after it was called.